extension/content.js

```diff
@@ -344,6 +344,21 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLinkId = 'refined-github-project-new-link';
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="${projectNewLinkId}">Add a project</a>`;
+	if ($(`#${projectNewLinkId}`).length === 0) {
```

Comment:
Re: checkbox: true that, this can prevent an unnecessary UI element for a feature which a user can decide to disable anyway. In this case we should extend the condition we use in the statement.

What about the case when a user has some projects already? Shall we still have the button shown in the Settings page?